src/components/portal/portal.tsx

```diff
@@ -73,16 +73,16 @@ export class Portal {
     @Watch('visible')
```

Comment:
Yay! This seems to work well, and it also seems to have the added benefit that menus with `open-direction="left"` no longer opens to the right and then jumps to the left the first time they are opened! 👍